documentation/configuration/locations.md

```diff
@@ -17,7 +17,7 @@ Unprefixed locations or locations starting with <code>classpath:</code> point to
 Locations starting with <code>filesystem:</code> point to a directory on the filesystem, may only contain SQL migrations and are only scanned recursively down non-hidden directories.
 
 ### Amazon S3
-Locations starting with <code>s3:</code> point to a bucket in AWS S3, may only contain SQL migrations, and are scanned recursively. They are in the format <code>s3:&lt;bucket&gt;(/optionalfolder/subfolder)</code>. To use AWS S3, the AWS environment variables <code>AWS_REGION</code>, <code>AWS_ACCESS_KEY_ID</code> and <code>AWS_SECRET_ACCESS_KEY</code> need to be set to the appropriate values for your S3 account.<br/>
+Locations starting with <code>s3:</code> point to a bucket in AWS S3, may only contain SQL migrations, and are scanned recursively. They are in the format <code>s3:&lt;bucket&gt;(/optionalfolder/subfolder)</code>. To use AWS S3, the [AWS SDK v2](https://mvnrepository.com/artifact/software.amazon.awssdk/services) must be included, and [configured](https://docs.aws.amazon.com/sdk-for-java/v1/developer-guide/credentials.html) for your S3 account.<br/>
```

Comment:
Do we want to mention that dependencies will also need to be included? I think dependencies is why users couldn't use S3 with the commandline, as it wasn't automatically getting them

Reply:
We probably should, though I'm not sure whether there's an exhaustive list of transitive dependencies (or whether that will change!)

Reply:
On the maven site you can see a list of dependencies for a specific artifact, maybe that's enough?